packages/remix/src/setupRemix.tsx

```diff
@@ -0,0 +1,23 @@
+import { createReactQueryHooks, createReactQueryHooksProxy } from '@trpc/react';
+import type { AnyRouter } from '@trpc/server';
+import { WithTRPCNoSSROptions, withTRPC } from './withTRPC';
+
+export function setupTRPC<TRouter extends AnyRouter>(
```

Comment:
Please rename to 

```suggestion
createTRPCRemix<TRouter extends AnyRouter>(
```

We're planning on aligning all packages to be `createTRPCXXXX`